Detectors/EMCAL/reconstruction/src/ClusterizerTask.cxx

```diff
@@ -0,0 +1,83 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file  ClusterizerTask.cxx
+/// \brief Implementation of the EMCAL cluster finder task
+
+#include "FairLogger.h"      // for LOG
+#include "FairRootManager.h" // for FairRootManager
+#include "EMCALReconstruction/ClusterizerTask.h"
+
+ClassImp(o2::emcal::ClusterizerTask);
+
+using namespace o2::emcal;
+
+//_____________________________________________________________________
+ClusterizerTask::ClusterizerTask(ClusterizerParameters* parameters) : mClusterizer(parameters->getTimeCut(), parameters->getTimeMin(), parameters->getTimeMax(), parameters->getGradientCut(), parameters->getDoEnergyGradientCut(), parameters->getThresholdSeedEnergy(), parameters->getThresholdCellEnergy())
+{
+}
+
+//_____________________________________________________________________
+/// \brief Init function
+/// Inititializes the digit reader & geometry in cluster finder
+void ClusterizerTask::init()
+{
+  // Initialize digit reader
+  if (!mDigitReader)
```

Comment:
also here, {} for 1-liners

Reply:
Done, I tried to find all occurrences in the commit.